.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,29 @@
+name: e2e
+on:
```

Comment:
```suggestion
on:
  push:
  - main
```

Reply:
Hm, adding this seemed to make it so it only runs on pushes to main for some reason. Taking a look.

Reply:
I'm under the impression that this is the correct behavior? In the future, we may want to update this push trigger to also trigger on `release-*` branches (in addition to the `main` branch) if we head towards that kind of release branch strategy.

Reply:
Hm, do we not want the E2E tests to run on every requested change incoming to the repo?

Reply:
There's a `pull_request` trigger that's dedicated towards triggering on pull request events. The `push` trigger just reacts when changes land on a branch or tags (and you would configure those branches/tags).

Reply:
Ah, turns out there was a syntax error that was causing it not to run correctly. Fixed this up!